Gopkg.lock

```diff
@@ -3,145 +3,260 @@
 
 [[projects]]
   branch = "master"
+  digest = "1:0c5485088ce274fac2e931c1b979f2619345097b39d91af3239977114adf0320"
```

Comment:
some of these options have been added due to using a newer version of dep from when this was initially generated